codegen-on-oss/examples/context_server_example.py

```
@@ -0,0 +1,115 @@
+    
+    if response.status_code != 200:
+        print(f"Error analyzing repository: {response.text}")
+        sys.exit(1)
+        
+    analysis_results = response.json()
+    print("Analysis complete!")
+    print(json.dumps(analysis_results, indent=2))
+
+    # Create a snapshot
+    print("\nCreating snapshot...")
+    response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)